tests/test_install_mender_sh.py

```diff
@@ -121,17 +128,32 @@ def test_client(
 
     @pytest.mark.usefixtures("setup_test_container_f")
     def test_connect(
-        self,
-        script_server,
-        setup_tester_ssh_connection_f,
-        mender_dist_packages_versions,
-        mender_version,
+        self, script_server, setup_tester_ssh_connection_f,
     ):
         localhost = self._get_localhost_ip(setup_tester_ssh_connection_f)
 
         setup_tester_ssh_connection_f.run(
             f"curl http://{localhost}:{SCRIPT_SERVER_PORT}/install-mender.sh | sudo bash -s -- mender-connect"
         )
 
-        setup_tester_ssh_connection_f.run("dpkg --status mender-client")
-        setup_tester_ssh_connection_f.run("dpkg --status mender-connect")
+        check_installed(setup_tester_ssh_connection_f, "mender-client")
+        check_installed(setup_tester_ssh_connection_f, "mender-connect")
+        check_installed(
+            setup_tester_ssh_connection_f, "mender-configure", installed=False
+        )
+
+    @pytest.mark.usefixtures("setup_test_container_f")
+    def test_configure(
+        self, script_server, setup_tester_ssh_connection_f,
+    ):
+        localhost = self._get_localhost_ip(setup_tester_ssh_connection_f)
+
+        setup_tester_ssh_connection_f.run(
+            f"curl http://{localhost}:{SCRIPT_SERVER_PORT}/install-mender.sh | sudo bash -s -- -c experimental mender-configure"
```

Comment:
Maybe add a similar `TODO` here to remove `-c experimental` once it is released.